x-pack/plugins/cases/server/client/alerts/update_status.test.ts

```diff
@@ -14,7 +14,7 @@ describe('updateAlertsStatus', () => {
     const savedObjectsClient = createMockSavedObjectsRepository();
 
     const casesClient = await createCasesClientWithMockSavedObjectsClient({ savedObjectsClient });
-    await casesClient.client.updateAlertsStatus({
+    await casesClient.client.updateStatus({
```

Comment:
I am wondering, how do we make the difference between updating an alert status vs the case status

Reply:
if it is an alert, it should be a function form the internalCaseClient, is that correct?

Reply:
The client has changed to `casesClient.cases.update` and `casesClientInternal.alerts.updateStatus`. I haven't update the tests yet :)